sirepo/sim_data/srw.py

```diff
@@ -76,6 +76,8 @@ def _compute_model(cls, analysis_model, *args, **kwargs):
             return 'multiElectronAnimation'
         if 'beamlineAnimation' in analysis_model:
             return 'beamlineAnimation'
+        if 'exportRsOpt' in analysis_model:
```

Comment:
Why is this an `in` and not `==`?

Reply:
copy-paste

Reply:
[815be41](https://github.com/radiasoft/sirepo/pull/4057/commits/815be4174fee8f9ee268720b7145f65971567670)